### DISCUSSION TITLE
Frigate LXC wont start after rebooot

### DISCUSSION THREAD

**Discussion by ForTheRush** [tteck/Proxmox#3665] [Help Needed]
Going through hell right now. had to flash a fresh proxmox install and when i tried to restore my frigate lxc from backup it kept throwing errors and wouldn't start so after dealing with that for an entire day i decided to just do a fresh install using the frigate script which worked great until i had to restart the container in order for it to see the devices i added in the proxmox gui. 

Well after a restart the lxc will not start. Now I've spent another full day trying to get a working container again. I tried editing the config. destroying and reinstalling. must have ran the script 10+ times today and i cant get it to start again after a reboot. but frigate can't see my coral tpu pci or usb after a new install and the script doesn't give you the option to not start the lxc before i can add the need devices.

It keeps throwing errors related to the devices that your script automatically adds as these don't exist. the devices im using have different names. if i delete those lines from the lxc.conf then it just continues with the autodev hooks errors. I'm pulling my hair out working in circles here to avail. not sure what else to do at this points.

errors on start:

` safe_mount: 1425 No such file or directory - Failed to mount "/dev/serial/by-id" onto "/usr/lib/x86_64-linux-gnu/lxc/rootfs/dev/serial/by-id"
safe_mount: 1425 No such file or directory - Failed to mount "/dev/ttyUSB0" onto "/usr/lib/x86_64-linux-gnu/lxc/rootfs/dev/ttyUSB0"
safe_mount: 1425 No such file or directory - Failed to mount "/dev/ttyUSB1" onto "/usr/lib/x86_64-linux-gnu/lxc/rootfs/dev/ttyUSB1"
safe_mount: 1425 No such file or directory - Failed to mount "/dev/ttyACM0" onto "/usr/lib/x86_64-linux-gnu/lxc/rootfs/dev/ttyACM0"
safe_mount: 1425 No such file or directory - Failed to mount "/dev/ttyACM1" onto "/usr/lib/x86_64-linux-gnu/lxc/rootfs/dev/ttyACM1"
run_buffer: 571 Script exited with status 17
lxc_setup: 3948 Failed to run autodev hooks
do_start: 1273 Failed to setup container "10010"
sync_wait: 34 An error occurred in another process (expected sequence number 4)
__lxc_start: 2114 Failed to spawn container "10010"
startup for container '10010' failed `

lxc config: 

`arch: amd64
cores: 4
dev0: /dev/dri/renderD128
dev1: /dev/bus/usb/004/003
dev2: /dev/apex_0
features: nesting=1
hostname: frigate-lxc
memory: 1024
net0: name=eth0,bridge=vmbr0,gw=192.168.1.1,hwaddr=BC:24:11:1C:54:B7,ip=192.168.1.8/24,type=veth
onboot: 1
ostype: debian
rootfs: local-lvm:vm-10010-disk-0,size=64G
swap: 512
tags: proxmox-helper-scripts
lxc.cgroup2.devices.allow: a
lxc.cap.drop:
lxc.cgroup2.devices.allow: c 188:* rwm
lxc.cgroup2.devices.allow: c 189:* rwm
lxc.mount.entry: /dev/serial/by-id  dev/serial/by-id  none bind,optional,create=dir
lxc.mount.entry: /dev/ttyUSB0       dev/ttyUSB0       none bind,optional,create=file
lxc.mount.entry: /dev/ttyUSB1       dev/ttyUSB1       none bind,optional,create=file
lxc.mount.entry: /dev/ttyACM0       dev/ttyACM0       none bind,optional,create=file
lxc.mount.entry: /dev/ttyACM1       dev/ttyACM1       none bind,optional,create=file
lxc.cgroup2.devices.allow: c 226:0 rwm
lxc.cgroup2.devices.allow: c 226:128 rwm
lxc.cgroup2.devices.allow: c 29:0 rwm
lxc.mount.entry: /dev/fb0 dev/fb0 none bind,optional,create=file
lxc.mount.entry: /dev/dri dev/dri none bind,optional,create=dir
lxc.mount.entry: /dev/dri/renderD128 dev/dri/renderD128 none bind,optional,create=file `

errors after removing those nonexistent devices :

`root@proxmox420:/etc/pve/lxc# pct start 10010
run_buffer: 571 Script exited with status 17
lxc_setup: 3948 Failed to run autodev hooks
do_start: 1273 Failed to setup container "10010"
sync_wait: 34 An error occurred in another process (expected sequence number 4)
startup for container '10010' failed
root@proxmox420:/etc/pve/lxc# `

**Comment by Porencephaly** [comment]
Did you ever solve this? I'm running into a similar issue with a jellyfin lxc
